### PR TITLE
Add at-since annotations up to 2.279

### PR DESCRIPTION
**Pull request by daniel-beck**
This notes additions from the following pull requests or commits:

* 2.253
  - https://github.com/jenkinsci/jenkins/pull/4888
* 2.257
  - https://github.com/jenkinsci/jenkins/pull/4517
* 2.258
  - https://github.com/jenkinsci/jenkins/pull/4829
* 2.259
  - https://github.com/jenkinsci/jenkins/pull/4922
  - https://github.com/jenkinsci/jenkins/pull/4550
* 2.260
  - https://github.com/jenkinsci/jenkins/pull/4027
* 2.265
  - https://github.com/jenkinsci/jenkins/pull/5041
  - https://github.com/jenkinsci/jenkins/pull/5026
* 2.266
  - https://github.com/jenkinsci/jenkins/pull/4848
* 2.267
  - https://github.com/jenkinsci/jenkins/pull/5015
  - https://github.com/jenkinsci/jenkins/pull/4274
* 2.272
  - https://github.com/jenkinsci/jenkins/pull/5121
* 2.275 and 2.263.2
  - https://github.com/jenkinsci/jenkins/commit/b1ca2849660798fd0f7fe8fda8e372cfb2ded621
  - https://github.com/jenkinsci/jenkins/commit/71d2ecf1a4e5303e80815eaa3935c4f2fa3d9104
* 2.279
  - https://github.com/jenkinsci/jenkins/pull/5220

### Proposed changelog entries

(Irrelevant)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/java/jenkins/install/SetupWizard.java
Comment (daniel-beck) [inline core/src/main/java/jenkins/install/SetupWizard.java:127]: I rephrased this one because it seemed ambiguous/unclear to me as it was before.

File: core/src/main/java/jenkins/model/Jenkins.java
Comment (daniel-beck) [inline core/src/main/java/jenkins/model/Jenkins.java:3906]: We don't generally note when something was deprecated, but it was in the original so I just filled in the version.

File: core/src/main/java/jenkins/security/apitoken/TokenUuidAndPlainValue.java
Comment (daniel-beck) [inline core/src/main/java/jenkins/security/apitoken/TokenUuidAndPlainValue.java:32]: It was unclear to me which of the version numbers is relevant here, so went into more detail for clarity.

**Comment by batmat** [comment]
Thanks Daniel!

Out of curiosity, did you use the `./update-since-todo.sh` script I had added, or did you do this manually? 
I'm happy to try and improve the script if this does not work :-)

**Comment by daniel-beck** [comment]
@batmat I've been a happy long-time user of your script 😉  I used it to identify locations and insert versions. I then manually looked up the commits and PRs to be able to reference them.

A deduplicated list of commit URLs written to stdout (any competent shell has one-click URL opening) would be a very useful addition, PRs are trivial to get from there. Looking at commits also helps ensure the generated versions are correct (https://github.com/jenkinsci/jenkins/pull/5276#discussion_r573305226 would not have been). If you want to be fancy, group by version number.

**Review by timja — APPROVED** [review]

**Comment by daniel-beck** [comment]
> A deduplicated list of commit URLs written to stdout (any competent shell has one-click URL opening) would be a very useful addition, PRs are trivial to get from there. Looking at commits also helps ensure the generated versions are correct ([#5276 (comment)](https://github.com/jenkinsci/jenkins/pull/5276#discussion_r573305226) would not have been). If you want to be fancy, group by version number.

Implemented in https://github.com/jenkinsci/jenkins/pull/5278

**Review by MarkEWaite — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**
